src/main/aciotest/main.c

```diff
@@ -16,7 +18,7 @@
 #include "util/log.h"
 
 static uint8_t aciotest_cnt = 0;
-static uint8_t bi2a_mode = 0;
+static uint8_t bi2a_mode = 255;
```

Comment:
Non-blocking remark to take away as a learning: An enum would improve readability here, e.g.

```
enum BI2A_MODE {
    BI2A_MODE_INVALID = 0,
    BI2A_MODE_SDVX = 1,
    BI2A_MODE_IIDX = 2,
};
```

If you want, you can follow up with another small refactoring PR to improve this. The current solution is fine though considering it's just two modes now and scoped to a single file.

Reply:
Sure, I'll do it later